jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/test/query/TestQueryHints.java

```diff
@@ -270,9 +333,13 @@ public PreparedStatementInvocationHandler(PreparedStatement stmt) {
         }
 
         public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
+            //Get the query timeout being set on the statement
+            //This value should be in seconds, since that is what the Statement expects
+            System.out.println(method.getName() + "INVOKED!!!!");
             if (method.getName().equals("setQueryTimeout") && proxy instanceof PreparedStatement) {
-                if(args.length > 0){
-                    TestQueryHints.setTimeout = (Integer)args[0];
+                System.out.println("INVOKED!!!!");
```

Comment:
this line should go away (or be sent to logger)

Reply:
Dang, I forgot to remove that... thanks for catching this